src/HeapDump/ICorDebugGCHeap.cs

```diff
@@ -1,1407 +0,0 @@
-﻿using Microsoft.Diagnostics.Runtime;
```

Comment:
@sharwell, was this used by Silverlight, or more generally?  If I recall correctly, this is a fallback path after `CLRMD`?

Reply:
Currently the file is only used if `Silverlight.DebugActiveSilverlightProcess` returns non-null. Porting this file to ClrMD 2 will be extremely difficult, if possible.

Reply:
Gotcha - just wanted to be sure.  Thanks.